src/intl/messages.js

```diff
@@ -91,6 +91,9 @@ export const messages: { [messageId: string]: MessageType } = {
   delete: 'Delete',
   description: 'Description',
   details: 'Details',
+  disconnect: { message: 'Disconnect', description: 'Text for disconnect button in noVNC console' },
+  disconectedConsole: 'Disconnected from Console',
+  disconectedConsoleInfo: 'The console has been diconnected. Press the \'Connect\' button to reconnect the console.',
```

Comment:
The console has been di**s**connected. Press the \'Connect\' button to reconnect the console.